instrumentation/opentelemetry-instrumentation-pika/src/opentelemetry/instrumentation/pika/utils.py

```diff
@@ -56,6 +63,10 @@ def decorated_callback(
             operation=MessagingOperationValues.RECEIVE,
         )
         with trace.use_span(span, end_on_exit=True):
+            try:
+                consume_hook(span, body, properties)
+            except Exception as e:
+                pass
```

Comment:
The `consume_hook` is written by the user, so if it raises I think it's ok to print the exception to the screen. Maybe add `logger.exception` with the exception?